documentation/source/library_reference.rst

```diff
@@ -346,6 +346,14 @@ The Scheduler
     :member-order: bysource
 
 
+Simulator
+---------
+
```

Comment:
Perhaps worth a note like I had in https://github.com/cocotb/cocotb/pull/1568/files#diff-7d491a84366bbf6a395ee614a5de3f21R365?

Reply:
Done.

Reply:
You don't appear to have pushed

Reply:
I was waiting to see if I could add more, but will push now.